public/js/constants.js

```diff
@@ -12,20 +13,22 @@ var spanishText = "Consciente de su patrimonio espiritual y moral, la Unión est
+    englishExpresiveSSML = "<speak>I have been assigned to handle your order status request.<express-as type=\"Apology\"> I am sorry to inform you that the items you requested are backordered. We apologize for the inconvenience.</express-as><express-as type=\"Uncertainty\"> We don't know when those items will become available. Maybe next week but we are not sure at this time.</express-as><express-as type=\"GoodNews\">Because we want you to be a happy customer, management has decided to give you a 50% discount! </express-as></speak>";
+	usEnglishSSML = "<p><s>Conscious of its spiritual and moral heritage <break time='300ms'/>, the Union is founded on the indivisible, universal values of <prosody rate='-15%'>human dignity, freedom, equality and solidarity.</prosody> It is based on the principles of democracy and the rule of law <break time='500ms'/>. </s> <s> It places the individual at the heart of its activities, <prosody rate='+15%'>by establishing the citizenship of the Union</prosody> and by creating an area of freedom, security and justice.</s></p>",
+    ukEnglishSSML = "<p><s>Conscious of its spiritual and moral heritage <break time='300ms'/>, the Union is founded on the indivisible, universal values of human dignity, <prosody rate='-15%'>freedom, equality and solidarity.</prosody> It is based on the principles of democracy and the rule of law <break time='500ms'/>. </s> <s> It places the individual at the heart of its activities, <prosody rate='+15%'>by establishing the citizenship of the Union</prosody> and by creating an area of freedom, security and justice.</s></p>",
+ 	germanSSML = "<p><s>In dem Bewusstsein ihres geistig-religiösen und sittlichen <phoneme alphabet='ibm' ph='.1R.0bIs'>Erbes</phoneme> <break time='300ms'/> gründet sich die Union auf die <prosody rate='-15%'>unteilbaren und universellen  Werte der Würde des Menschen, der Freiheit, der Gleichheit und der Solidarität.</prosody> Sie beruht auf den Grundsätzen der Demokratie und der Rechtsstaatlichkeit<break time='1s'/>. Sie stellt den Menschen in den Mittelpunkt ihres <phoneme alphabelt='ibm' ph='.1hAn.0d@lns'> Handelns</phoneme>, <prosody rate='+10%'>indem sie die Unionsbürgerschaft und einen Raum der Freiheit, der Sicherheit und des Rechts begründet.</prosody></s></p>",
     italianSSML = "Italian SSML is not presently supported",
     japaneseSSML = "Japanese SSML is not presently supported",
     brazilianPortugueseSSML = "Brazilian Portuguese SSML is not presently supported";
-
+	
 window.SPEECH_SYNTHESIS_VOICES = {
     voices: [
     {
         "name": "en-US_AllisonVoice", 
         "language": "en-US", 
         "customizable": true, 
-        "gender": "female", 
+        "gender": "female, expressive", 
```

Comment:
@daniel-bolanos sent out an email to Juraj Figura & Arthur Kantor,asking for confirmation on what the REST API will look like. We'll go from there.